nginx.conf

```diff
@@ -0,0 +1,29 @@
+events {
+    worker_connections 1024;
```

Comment:
real call to ms-auth, handling for 401,406. And set response headers from auth to request headers